base/card-api.gts

```diff
@@ -990,6 +975,70 @@ export class Box<T> {
     return newChildren;
   }
 
+  // trying to determine the card from the outside of the box destabilizes otherwise stable
+  // arrays due to glimmer invalidation in our field getters. I'm breaking the rule where
+  // boxes aren't cognizant of cards to work around this situation
+  get card() {
+    return typeof this.value === 'object' && this.value != null && isBaseCard in this.value
+      ? Reflect.getPrototypeOf(this.value as unknown as Card)!.constructor as typeof Card 
+      : undefined;
+  }
```

Comment:
this was an important finding that allowed me to determine the cards for field values without destabilizing models